<!DOCTYPE html>
<html>
<head>
	<meta CHARSET="utf-8">
	<meta name="author" CONTENT="miwinfo.com" />
    <meta name="viewport" CONTENT="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
	<link HREF="favicon.ico" REL="shortcut icon">
	<title>HUST-CHINA</title>
    
<link REL="stylesheet" HREF="css/css.css">
<link REL="stylesheet" HREF="css/response.css">
<link REL="stylesheet" HREF="css/pc.css">

<script SRC="javascript/jquery-1.4.min.js"></script>
<script SRC="javascript/jquery-3.3.1.js"></script>
<script SRC="javascript/js.js"></script>
<script SRC="javascript/jquery-1.4.2.min.js"></script>
<script SRC="javascript/jquery-2.0.3.min.js"></script>
<script TYPE="text/javascript">
</script>

<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
</head>
<body>

<!--home-->
<div>
     <div id="firstpage"> 
     <img CLASS="hidden" id="firstimg" SRC="images/BK.jpg"/>
     <div id="klam1" CLASS="display">Butterfly Effect</div>
     <div id="klam2" CLASS="display">HUST-CHINA</div>
     <div CLASS="noor one"></div>
     <div CLASS="noor two"></div>
     </div>
     
     <div id="thirdpage">
         <div STYLE=" width:100%; height:20px;"></div>
         <div id="abstract"><h1> ABSTRACT</h1></div>
         <div id="abstract_text"><h2>Azobenzene is a photoresponsive compound which contains two symmetrical benzenes like a pair of butterfly wings. It isomerizes from trans to cis with light switching from the visual to UV, like a butterfly waving its wings, which generates the Butterfly Effect. Once it's inserted between adjacent nucleobases, the tran-to-cis isomerization triggers DNA unwinding as light changes. The same magic also applies to derivatives of azobenzene, S-DMAzo. Based on their outstanding properties, we design a light-driven nanomachine. The nanomachine consists of a ring as walker and a rod as track.As light of two different wavelengths switches, the doors made of two kinds of photoresponsive DNA strands on the rod open in turns so the ring carrying cargo moves along the track step by step. In a nutshell, this photo-driven nanomachine absolutely prevents the contamination compared with fuel-motivated nanomachines and also provides a new view on oriented molecule transport.</h2></div>
         <div STYLE=" width:100%; height:20px;"></div>
     </div>
     <div id="secondpage">
     <div STYLE=" width:100%; height:20px;"></div>
         <div><h1> VIDEO</h1></div>
	 <center>
           <div style="width:100%; "></div>
		  <a href="javascript:;" class="video " id="video"><img src="images/screen.jpg"/></a>
	  <div class="video-btn" id="video-btn">
			<div class="video-area" id="video-area"></div>
            <a id="video-shut"><img src="images/close.png"/></a>
		</div>
		<div id="shadow"></div>
	 </center>
     </div>

</div>


<div STYLE="width:100%; height:3px; background-color:black;"></div>


<!--main body-->

<div id="box">
<div id="inbox">




<!--project-->
<div id="project">
    <div STYLE="width:100%; height:40px;"></div>
<div><h1>PROJECT</h1></div>
    <div STYLE="background:linear-gradient(to right,#D55757,#FFF); width:80%; height:3px; margin-left:50px;"></div>
<h2 id="p1">Background</h2>
            <div CLASS="content">DNA origami plays an indispensable role in assembling spectacular micro-architectures and constructing molecular machines in nano world. By folding a long single strand (scaffold, 7249bp, as building block) and combining it with a series of well-designed oligonucleotide strands, DNA strands self-assemble and construct highly complicated nanostructure under controlled environment according to the principle of complementary base pairing<div CLASS="reftextp1 reftext">[1]</div>.</div>
            <div CLASS="eimg50"><img SRC="images/pro/p1.png"></img></div>
            <div CLASS="pfig">Figure 1</div>
            <div CLASS="content">Traditionally, nanomachines have been motivated by the input of chemical energy, for example, adding fuel strands. Competitive strands displacement reaction between fuel chains and staples (combined with DNA scaffold) powers the movement of nanomachines<div CLASS="reftextp2 reftext">[2]</div>. However, such control method remains many disadvantages. First of all, the utilization efficiency is rather low. Strands displacement basically needs a long responsive time in reaction system. Also, the double-strand waste accumulates as the reaction goes on, which lowers the concentration of fuel chains gradually, bringing rate reduction to subsequent movements and inevitable chemical contamination.</div>
            <div CLASS="eimg40"><img SRC="images/pro/p2.png"></img></div>
            <div CLASS="pfig">Figure 2</div>
    
<h2 id="p2">Our idea</h2>
           <div CLASS="content">Here, we design a photo-controlled molecular motor, aiming to adopt a new power-input approach that regulates the movement of nanomachine.</div>
           <div CLASS="content">According to the research of Asanuma et al, a chemical called Azobenzene(Azo) and its derivative 2',6'-Me-Azo(S-DMazo) have excellent photo-responsive properties<div CLASS="reftextp3 reftext">[3]</div>. Moreover, in view of the fact that the space size of Azo and its derivative is very close to the base in nucleic acid, it is possible to wedge Azo or its derivative between two nucleotides in DNA single strand and regulate the hybridization or dissociation of duplex on exposure to different light conditions. As the figure 3 shows, duplex wedged with Azo dissociates at the wavelength of 340nm when the strand with S-DMAzo hybridizes. On the contrary, the duplex wedged with S-DMAzo unwounded under the light of 390nm meanwhile strand wedged with Azo pairs.</div>
           <div CLASS="eimg40"><img SRC="images/pro/p3.png"></img></div>
           <div CLASS="pfig">Figure 3 </div>
           <div CLASS="content">Our nanomachine consists of a track with two rods and a ring made of DNA origami. As the figure 4 shows, two kinds of strands (marked in two colors) are attached to certain positions on the rod and ring. With photoswitches under different wavelengths of light, the ring moves from one end of the rod to the other. This nanomachine solves chain contamination problem and improves the reaction efficiency in system. Like a butterfly flaps it's wings, Azo isomerizes so that the duplex wedged with Azo dissociates and the ring steps along the rod, moving through the block in succession like kinesin<div CLASS="reftextp4 reftext">[4]</div>, which inspires us to mimic the distance-reliant biochemical process in vitro and measure it' s accessible length in nanoscale. The oriented-moving ring opens a new door in the field of molecule transport.</div>
           <div CLASS="eimg80"><img id="pimg4_1" src="images/des/d12.gif"></img><img id="pimg4_2" SRC="images/pro/p5.gif"></img></div>
           <div CLASS="pfig">Figure 4</div>
           <div CLASS="content">The isomerization of Azo and S-DMazo can be compared to the process of two beautiful butterflies waving their wings. Furthermore, as these compounds inserted in DNA sequence, photo-isomerization between trans and cis forms leads to the hybridization and dissociation in dsDNA just like a butterfly fluttering in Rio de Janeiro could change the weather in Chicago. The butterfly effect sensitively emphases the unexpected role initial conditions plays in which a small change in one state of a deterministic nonlinear system may bring about large differences in a later state. Just like our nanomachine, the small change in conformation can result in nanoscale movement. That's the reason why we name our project  <span>Butterfly Effect.</span></div>
            <div CLASS="eimg40"><img SRC="images/pro/p6.png"></img></div>
            <div CLASS="pfig">Figure 5</div>
    
    
<div><h2 id="p3">Achievements</h2></div>
            

            <div CLASS="achword">
            <span id="ach1"><a HREF="#d3">DNA Design</a></span>
            <span id="ach2"><a HREF="#d2">Chemical Reaction</a></span>
            <span id="ach3"><a HREF="#e2">Characterization</a></span>
             <span id="ach4"><a HREF="#e1">Synthesis</a></span>
            <span id="ach5"><a HREF="#e10">Function test</a></span>
            <span id="ach9">Add ordinary staples</span>
            <span id="ach10">Wedged into DNA strands</span>
            <span id="ach11">Annealing</span></div>
            
           
            <div CLASS="eimg60"><img SRC="images/pro/p7.png"></img></div>
           
<div><h2 id="p4">Future work</h2></div>
           <div CLASS="content">In order to test the nanomachine's ability to transport molecules, we plan to test and verify a reaction process. One typical cascade reaction is between glucose oxidase<div CLASS="reftextp5 reftext">[5]</div>(GOx) and horseradish peroxidase<div CLASS="reftextp6 reftext">[6]</div>(HRP). GOx catalyzes the oxidation of glucose by oxygen and generates gluconic acid with hydrogen peroxide, while HRP reduces the hydrogen peroxide to H2O and turns ABTS2- to ABTS.  HRP is modified to the ring and GOx to the rod. As our nanomachine gives a ride to these enzymes, HRP gets closer to GOx. Therefore, reaction rate can be demonstrated quantitatively by factor consisting of rod length and the frequency of UV and Vis operation. Hopefully, our design could provide a more accurate way to describe the process of enzymes reaction.</div>
           <div CLASS="eimg40"><img SRC="images/pro/p8.png"></img></div>
           <div CLASS="pfig">Figure 6 </div>
           
           
<div class="reference">
           <div class="rtitle">REFERENCE</div>
           <div class="rword">1.	Rothemund PW. Folding DNA to create nanoscale shapes and patterns. Nature. 2006;440(7082):297-302.</div>
           
           <div class="rword">2.	Shin J-S, Pierce NA. A Synthetic DNA Walker for Molecular Transport. Journal of the American Chemical Society. 2004;126(35):10834-5.</div>
           <div class="rword">3.	Nishioka H, Liang X, Kato T, Asanuma H. A photon-fueled DNA nanodevice that contains two different photoswitches. Angew Chem Int Ed Engl. 2012;51(5):1165-8.</div>
           <div class="rword">4.	Bhabha G, Johnson GT, Schroeder CM, Vale RD. How Dynein Moves Along Microtubules. Trends Biochem Sci. 2016;41(1):94-105.</div>
           <div class="rword">5.	Wong CM, Wong KH, Chen XD. Glucose oxidase: natural occurrence, function, properties and industrial applications. Applied Microbiology and Biotechnology. 2008;78(6):927-38.</div>
           <div class="rword">6.	Carlsson GH, Nicholls P, Svistunenko D, Berglund GI, Hajdu J. Complexes of Horseradish Peroxidase with Formate, Acetate, and Carbon Monoxide. Biochemistry. 2005;44(2):635-42.</div>
           
        
</div>           
           
 </div>          
           
           
           







<!--design-->
<div id="design">
    <div STYLE="width:100%; height:30px;"></div>
<div><h1>DESIGN</h1></div>
    <div STYLE="background:linear-gradient(to right,#D55757,#FFF); width:80%; height:3px; margin-left:50px;"></div>
    
    
    <div><h2 id="d1">Compound</h2></div>
            <div CLASS="content">Azobenzene contains two benzene rings bridged by a N=N double bond. It has both cis and trans conformation.Attractively, Azobenzene(Azo) and its derivative 2',6'-Me-Azo(S-DMazo) have excellent photo-responsive properties. In visual light, Azo remains in trans structure while it transforms to cis structure once exposed to ultraviolet （UV，λ=340 nm). The form of Azo changes when irradiation condition switches. So does it with SD-Mazo.  </div> 

            <div CLASS="content">With modification of two methyl groups at the ortho positions of azobenzene, a methylthio and a carboxylate group at the para positions, S-DMazo (2',6'-Me-Azo),  is synthesized. Because carboxyl group pulls the electron and the alkyl group weakly pushes it, which is called a push–pull (or push–push) process, the absorption maximum shifts to the visible region (λ=390 nm). </div>
            <div CLASS="eimg60"><img id="dimg1_1" SRC="images/des/d1_1.gif"></img><img id="dimg1_2" SRC="images/des/d1_2.gif"></img></div>
            <div CLASS="pfig">Figure 1</div>
            <div CLASS="content">There usually exist two mechanisms to describe the thermal isomerization process: inversion and rotation. The inversion route proceeds through a transition where one of the nitrogen atoms is sp hybridized, whereas the rotation route involves in disruption of a nitrogen–nitrogen π bond and rotation around the remaining σ bond. The sp-hybridized nitrogen bond is stable in nonpolar solvent due to its hydrophobicity while the rotation route is stabilized in polar system owing to its N-N single bond. Therefore, the solvent dependence helps select the real mechanism-----inversion<div CLASS="reftext reftextd1">&nbsp;[1]</div>.</div>
            <div CLASS="eimg40"><img SRC="images/des/d2.png"></img></div>
            <div CLASS="pfig">Figure 2 </div>
            <div CLASS="content">Meanwhile, the substitution for Azo with two ortho methyl groups remarkably hinders the thermal isomerization from trans to cis. At the transition state, lone pair electrons in the p orbit in Nα atom cause steric hindrance with the methyl groups on the twisted distal benzene rings, which markedly retards the inversion to the trans form.</div>
            <div CLASS="eimg40"><img SRC="images/des/d3.png"></img></div>
            <div CLASS="pfig">Figure 3 </div>
            <div CLASS="content">Therefore, Tm of S-DMazo (2',6'-Me-Azo) in trans form increases while the one in cis form drops, leading to the multiplication of △Tm and giving a three-times larger change in △Tm than the unmodified form. </div>
            <div CLASS="content">  In conclusion, it shows great photoregulatory ability and thermal stability in Azo and S-DMazo.</div>
    <div><h2 id="d2">Compound wedged in DNA</h2></div>
            <div CLASS="content">It is reported that the size of the Watson–Crick base-pair in this case is around 11*10^(-9)m, whereas for unmodified azobenzene it is 11–12*10^(-9)m, whereas for unmodified azobenzene it is 11–12*10^(-9)nm. So it is possible to wedge Azo and its derivative into DNA sequence to replace original bases. Several research groups have utilized azobenzene to control duplex formation. Alternating UV/vis illumination results in cis-to-trans isomerization of Azo and its derivatives, which trigger the dissociation of DNA duplex.</div>
            <div CLASS="eimg40"><img SRC="images/pro/p3.png"></img></div>
            <div CLASS="pfig">Figure 4 </div>
            <div CLASS="content">Moreover, the properties changes if Azo is modified by different functional groups, so it does with the peaks of its photoresponsive wavelength<div CLASS="reftext reftextd2">&nbsp;[2]</div>. Compared with 340nm maximum absorption wavelength in Azo, absorption property of S-DMazo shifts to 390nm due to modification of two methyl groups at the two ortho positions of azobenzene. The Photo-responsive wavelength for Azo is 340 nm where it remains in cis but steric hindrance between strands increases, causing DNA double strands unpaired. Under 390 nm irradiation, Azo in DNA keeps in trans form due to the space double helix distortion force, counteracting the base stacking force between Azo and adjacent bases so the duplex strands remains unchanged. In a word, the photoresponsive wavelengths of the corresponding derivatives vary with different functional groups modified on azobenzene. </div>
            <div CLASS="content">Therefore, we take advantage of these excellent properties to design a light-controlled nanomachine. </div>
             
    <div><h2 id="d3">Structure: DNA ring</h2></div>
            <div CLASS="content">As shown in Figure 5, the DNA track is composed of two identical rods connected in U shape and the DNA ring
  moves freely on either of the rods. This design helps the ring get on the rod with a better chance. Three-view drawings exhibits DNA rod in three aspects, respectively. </div>
            <div CLASS="eimg50"><img SRC="images/des/d5.png"></img></div>
            <div CLASS="pfig">Figure 5 </div>
            <div CLASS="content">As shown in Figure 6, two kinds of ssDNAs（P1-1/2，P2-1/2 ) are attached to the ring as legs, which can bind to specific anchor strands along DNA rod . The specific DNA sequence is also shown in figure 6. The sequences in different color represent their functions. The sequences marked in black fix the leg in ring. The sequences in blue and red are designed to hybridize with anchor strands on rods meanwhile the purple parts don't pair with any sequence and it's especially designed to elongate door strands and provide large convenience to lock the doors. </div>
            <div CLASS="eimg50"><img SRC="images/des/d6.png"></img></div>
            <div CLASS="pfig">Figure 6</div>
            <div CLASS="content">The detailed caDNAno design pictures are shown in figure 7.Click the second picture in Figure 7 for original caDNAno design.<a class="download" href="images/ring.csv">(Download all staple sequences of the ring staples)</a></div>
            
            <div CLASS="eimg50"><img id="dimg6_1" SRC="images/des/d7_4.png"></img><img id="dimg6_2" CLASS="pimg" SRC="images/des/d6_2.png" STYLE="width:100; height:100;"></img></div>
            <div id="outerdiv" STYLE="position:fixed;top:0;left:0;background:rgba(0,0,0,0.7);z-index:2;width:100%;height:100%;display:none;">
             <div id="innerdiv" STYLE="position:absolute;">
               <div id="imgbox" STYLE="overflow-y:scroll; overflow-x:hidden; width:800px; height:600px;">
                      <img id="bigimg" STYLE="border:5px solid #fff;"/>
               </div>
             </div>
           </div>
           <div CLASS="pfig">Figure 7</div>
            <div CLASS="content">The result of CanDo are shown in Figure 8</div>
            <div CLASS="eimg80"><img  id="dimg7_1" SRC="images/des/d7_1.gif"></img><img  id="dimg7_2"  src="images/des/d7_2.gif"></img><img   id="dimg7_3" SRC="images/des/d7_3.gif"></img></div>
            <div STYLE="width:100%; height:20px;"></div>
            <div CLASS="pfig">Figure 8</div>
           
            
    <div><h2 id="d4">Structure: DNA track</h2></div>
            <div CLASS="content">As shown in Figure 9, the DNA track is composed of two identical rods connected in U shape and the DNA ring moves freely on either rod. This design helps the ring get on the rod with a better chance. Three-view drawings pictures exhibits DNA rod in three aspects, respectively. </div>
            <div CLASS="eimg50"><img SRC="images/des/d8.png"></img></div>
            <div CLASS="pfig">Figure 9 </div>
            <div CLASS="content">As shown in Figure10, The linking strands between two rods can be categorized as anchor strands and door strands based on their functions that anchor strands contribute to trapping the moving ring, meanwhile the door strands, as their names suggest, block the ring's drawback.</div>
            <div CLASS="content">The door and anchor strands wedged with Azo and S-DMAzo  separated uniformly on the rods. 	The switches of irradiation wavelengths between 340nm and 390nm make two kinds of strands (in blue and red colors respectively) lock alternately in order to regulate the movements of the ring on the track.</div>
            <div CLASS="eimg50"><img SRC="images/des/d9.png"></img></div>
            <div CLASS="pfig">Figure 10</div>
            <div CLASS="content">The detailed caDNAno design pictures are shown in figure 11.Click the second picture in Figure 11 for original caDNAno design.<a class="download" href="images/ring.csv">(Download all staple sequences of the track staples)</a></div>
            <div CLASS="eimg50"><img id="dimg11_1" src="images/des/d10_4.png"></img><img id="dimg11_2" class="pimg" src="images/des/d11_2.png"></img></div>
            <div CLASS="pfig">Figure 11</div>
            <div CLASS="content">The result of CanDo are shown in Figure 12</div>
            <div CLASS="eimg80"><img  id="dimg10_1" SRC="images/des/d10_1.gif"></img><img  id="dimg10_2"  src="images/des/d10_2.gif"></img><img  id="dimg10_3" SRC="images/des/d10_3.gif"></img></div>
            <div STYLE="width:100%; height:20px;"></div>
            <div CLASS="pfig">Figure 12</div>


    <div><h2 id="d5">Movement</h2></div>
            <div CLASS="content">In order to forestall the ring to fall off the rod, we adopt another kind of door at the very beginning as figure 13 shows. It consists of two single strands linking to both ends of two rods as lock strands. After successfully incubating rings and rods together, lock strands are added later to close off the entrance.</div>
            <div CLASS="eimg40"><img SRC="images/des/d11.gif"></img></div>
            <div CLASS="pfig">Figure 13 </div>
            <div CLASS="content">(P1' refers to the strands in blue, P2' is in red on the rod as door )</div>
            <div CLASS="content">When illuminating 390nm light, Azo keeps in trans form while S-DMAzo remains in cis form so Azo-tethered strands can form the duplex but S-DMAzo-tethered strands cannot. The ring binds to the P1' anchor(Fig.14). At first, the ring moves aimlessly along the track in dynamics. Later, it will be entrapped into P2' anchor to form duplex and has no way to move in previous direction, exhibiting the movements of the ring from the first door to the next. When alternating the light to 390 nm again, the ring faces the same situation and it binds the P1'instead. When illuminating 340nm light, instead, the situation interchanges exactly between S-DMAzo and Azo. The ring will leave P1' and bind to P2' anchor.With consequent photoswitching, the ring moves in one direction continuously. </div>
            <div CLASS="eimg40"><img SRC="images/des/d12.gif"></img></div>
            <div CLASS="pfig">Figure 14</div>
            <!--<div class="content">By constantly switching the lighting, the walker moves in one direction continuously. The whole movement is shown in figure 17 clearly. </div>
            <div class="img_box3_2"><img src="images/figure4.gif"></img></div>
            <div class="fig_box3">Figure 17: </div>-->
            <div CLASS="content">Fluorescence resonance energy transfer (FRET) occurs when the distance between suitable donor and receptor decreases by no more than 10nm. Then donor generates a visual reduction of emission intensity on the spectrum and receptor emits stronger fluorescent signal. Donor FAM and Cy3 are employed on the ring and corresponding receptor DAB is fixed on the track. However, Considering the signal interfere between two fluorophores, the FRET experiment needs to be finished in different groups. Each group is designed to demonstrate movements of the ring in a limited range for one step via a single pair of fluorophore. The results collected can finally confirm the whole movements along the track.</div> 
            
            <div CLASS="eimg50"><img SRC="images/des/d13.png"></img></div>
            <div CLASS="pfig">Figure 15</div>
     <div><h2 id="d6">Model</h2></div>
            <h3>1. &nbsp;&nbsp;&nbsp;Background</h3>
            <div CLASS="content">In order to describe our model in a mathematical way, give some suggestions on experiments and apply our design into a practical use, we set up series of models to depict our architecture. Firstly, we focus on the chemical process of ring loading on rod. This model is purposed to give us a suggestion on proportion of rings and rods while incubation. Secondly, we try to use model to describe DNA damage under UV irradiation. Azobenzene can be used to render our DNA machine photo-controlled which means specific wavelength UV condition is essential. However, UV spectra poses a potential issue since DNA can absorb UV light and undergo unwanted photochemical changes<div CLASS="reftext reftextm1">&nbsp;[3]</div>. Such changes may bring alternation in conformation of DNA architecture. It is necessary to make description on photo damage. Third, we try to apply our structure to enzyme distance controling, DNA storage unit. Thus, kinematics description is also an important aspect we need to focus on. With mathematic solid we manipulate our design in an appropriate manner.</div>
            <h4>1.1 &nbsp;&nbsp;&nbsp;Basic questions</h4>     
            <div CLASS="content">Based on background, we then rise three basic and direct questions </div>
            <h5>1.1.1&nbsp;&nbsp;Dynamic of combination process</h5>    
                <div CLASS="content">How to describe loop-to-rod process linking two tiles through single strands DNA strands. This is attempt to answer the question what's the ring to rods proportion should we set in incubation system to make sure that most of the rings can efficiently be assembled on rods. </div>
            <h5>1.1.2&nbsp;&nbsp;Photo-damage under UV irradiation</h5>
            <div CLASS="content">Ultraviolet light can damage DNA molecule, but light absorption of DNA molecule is different from one wavelength to another<div CLASS="reftext reftextm1">&nbsp;[1]</div>. Therefore, how to measure the damage of DNA element while irradiating to the moving ring need to be considered quantitatively.</div>
            <h5>1.1.3&nbsp;&nbsp;Kinematics of architecture</h5>
            <div CLASS="content">For the whole process of the ring moving on rod, it should be pointed out that the power pushing the ring moving forwards is not striving from the light, but is caused by the molecular thermal motion of ring itself (Brownian motion). Meanwhile the single strand linking up and down rods and down rod only plays a blocking role. Thereby, the motion of the ring on the track also needs to be considered. Corresponding models are used for prediction.</div>
            <h4>1.2 &nbsp;&nbsp;&nbsp;Assumption </h4>
            <div CLASS="content">1)The model considered here is a molecular system with low frequency vibration, so the shape of the molecule is chosen as the main determinant rather than the interaction between atoms. Under this assumption, the motion of a molecular system can be considered as a simple harmonic, and its potential energy can be applied to Hookean springs <div CLASS="reftext reftextm1">&nbsp;[3]</div>.</div>
            <div CLASS="content">2)During incubation, the single strands links on the rods are quite sufficient, so their concentration can be regarded as constant, and the single strand pairing reaction on the rods is considered as a first-order reaction, so our ring-on-rod reaction can be described by factor J.</div>
            <div CLASS="content">3)The single strand of rod in the same ring is considered to act simultaneously when the rod acts<div CLASS="reftext reftextm6">&nbsp;[8]</div>. There is no interaction among the strands complementary.</div>
            <div CLASS="content">4)All reaction constants and reactions do not take into account size of the DNA architecture, and each DNA single-stranded connection region has the same probability of connection and the same probability of shedding</div>
            <div CLASS="content">5)The intensity of light received by each location of DNA origami is the same. Moreover, DNA will not produce other allosteric effects<div CLASS="reftext reftextm2">&nbsp;[4]. </div>
</div>
            <h3>2. &nbsp;&nbsp;&nbsp;Model setup</h3>
            <h4>2.1&nbsp;&nbsp;&nbsp;Dynamic of combination process</h4>
            <div CLASS="content">The reaction on the upper ring can be regarded as a reversible first order reaction (first order reaction).</div>
            <div CLASS="function1"><img SRC="images/simu/s1.png"></img></div>

            <div CLASS="content">The rate constants of reversible reactions can be expressed by Ka and Kd, and the J factor represents a constant product of Kansas which together represents the bimolecular reaction rate. The ratio of S products can be expressed as j*Ka[Rn].Therefore, from the reaction described above, we can obtain the concentration relationship of the product at the unit time we need. We can have the following equations</div>
            <div CLASS="function1"><img SRC="images/simu/s2.png"></img></div>
            <div CLASS="function1"><img SRC="images/simu/s3.png"></img></div>

            <div CLASS="content">For a reaction in the same system we can regard concentration as the number of chains in the reaction, so multiplying the concentration variables on both sides by the equivalent volume is available.</div>
            <div CLASS="function1"><img SRC="images/simu/s4.png"></img></div>

            <div CLASS="content">When the ring is on the rod, the number of pairings that we need varies with time by integrating the two sides together </div>
            <div CLASS="function1"><img SRC="images/simu/s5.png"></img></div>
            <div CLASS="function1"><img SRC="images/simu/s6.png"></img></div>

            <h4>2.2&nbsp;&nbsp;&nbsp;Photo-damage under UV irradiation </h4>
            <div CLASS="content">In this section, we take purine dimerizing as the only effect to damage DNA strands. The effect which can cut off part of chemical bonds will not be considered. According to the empirical formula measured in this paper<div CLASS="reftext reftextm4">&nbsp;[4]</div>.</div>
            <div CLASS="function1"><img SRC="images/simu/s7.png"></img></div>
            <div CLASS="function1"><img SRC="images/simu/s8.png"></img></div>


            <div CLASS="content">Where N represents the total BP number, V represents the volume under illumination, which is constant 1.24*104 1/(M*cm)<div CLASS="reftext reftextm1">&nbsp;[2]</div>. We have the corresponding absorbance values for each bp and the possible energy of purine two aggregation.</div>
            <div CLASS="function1"><img SRC="images/simu/s9.png"></img></div>

            <h4>2.3&nbsp;&nbsp;&nbsp;Kinematics of architecture</h4>
            <div CLASS="content">The direct connection between the ring and the rod is also considered. However, the Gillespie reaction-diffusion equation can be used for the motion of the ring. The downstream reaction is interacted with upstream reaction. First, set the corresponding probability density function<div STYLE="display:inline;"><img SRC="images/simu/s10.png"></img></div>,<div STYLE="display:inline; vertical-align:middle;"><img SRC="images/simu/s11.png"></img></div>. Which can be considered as reaction under condition<div STYLE="display:inline; vertical-align:middle;"><img SRC="images/simu/s12.png"></img></div> at T. whereas, the next reaction will occur at<div STYLE="display:inline; vertical-align:middle;"><img SRC="images/simu/s13.png"></img></div>.</div>
            <div CLASS="content"><div STYLE="display:inline; vertical-align:middle;"><img SRC="images/simu/s14.png"></img></div> this reaction is a duplex inserted with azo benzene.</div>
            <div CLASS="content"><div STYLE="display:inline; vertical-align:middle;"><img SRC="images/simu/s15.png"></img></div>this reaction is a duplex inserted with DM azo benzene</div>
            <div CLASS="content">First, a theorem is introduced for the model.</div>
            <div CLASS="content">Theorem 2.1 if <div STYLE="display:inline;"><img SRC="images/simu/s16.png"></img></div>, then we have</div>
            <div CLASS="function1"><img SRC="images/simu/s17.png"></img></div>
            <div CLASS="function1"><img SRC="images/simu/s18.png"></img></div>


            <div CLASS="content">Thus apply theorem on our system. We have</div>
            <div CLASS="function1"><img SRC="images/simu/s19.png"></img></div>
            <div CLASS="function1"><img SRC="images/simu/s20.png"></img></div>


            <div CLASS="content">Then we have distribution function by integrating</div>
            <div CLASS="function1"><img SRC="images/simu/s21.png"></img></div>

            <div CLASS="content">Similarly, we can have<div STYLE="display:inline;"><img SRC="images/simu/s22.png"></img></div> distribution function <div STYLE="display:inline;"><img SRC="images/simu/s23.png" ></img></div>.</div>
            <div CLASS="function1"><img SRC="images/simu/s24.png" ></img></div>

            <h3>3. &nbsp;&nbsp;&nbspProblems </h3>
            <div CLASS="content">Big molecular reaction is a very complicated system accompanying with many random movements. Due to these movement, some of interactions among big molecules are unable to anticipate leaving along modeling it. </div>
            <div CLASS="content">1)We consider Steric hindrance as an integrated variant containing inside the probability function other than taking it as a variant.</div>
            <div CLASS="content">2)We are unable to setup a function to analog brown motion for it is not accurate to consider that the ring has the same odds to move forwards or backwards.</div>
</div>

<div class="reference">
           <div class="rtitle">REFERENCE</div>
           <div class="rword">1.	Bandara HMD, Burdette SC. Photoisomerization in different classes of azobenzene. Chemical Society Reviews. 2012;41(5):1809-25.</div>
           <div class="rword">2.	Nishioka H, Liang X, Kato T, Asanuma H. A photon-fueled DNA nanodevice that contains two different photoswitches. Angew Chem Int Ed Engl. 2012;51(5):1165-8.</div>
           <div class="rword">3.MECHANICS AND DYNAMIC BEHAVIORS OF DNA ORIGAMI docs.lib.purdue.edu/dissertations/AAI10268974/</div>
           <div class="rword">4.Alireza Sepehri. A mathematical model for DNA[J]. International Journal of Geometric Methods in Modern Physics, 2017, 14.</div>
           <div class="rword">5.López-Blanco J R, Chacón P. New generation of elastic network models[J]. Current Opinion in Structural Biology, 2016, 37:46-53.</div>
           <div class="rword">6.Zarai Y, Margaliot M, Tuller T. A deterministic mathematical model for bidirectional excluded flow with Langmuir kinetics[J]. Plos One, 2017, 12(8):e0182178.</div>
           <div class="rword">7.Garel T, Monthus C, Orland H. A simple model for DNA denaturation[J]. Epl, 2001, 55(1):132-138.</div>
           <div class="rword">8.You M, Chen Y, Zhang X, et al. An Autonomous and Controllable Light-Driven DNA Walking Device[J]. Angewandte Chemie International Edition, 2012, 124(10):2507-2510.</div>
</div>
<!--experiment-->
<div id="experiment">
    <div STYLE="width:100%; height:40px;"></div>
<div><h1>EXPERIMENT</h1></div>
    <div STYLE="background:linear-gradient(to right,#D55757,#FFF); width:80%; height:3px; margin-left:50px;"></div>
<h2 id="e1">Protocol</h2>
            <h3>1. &nbsp;&nbsp;&nbsp;The extraction and verification of scaffold</h3>
            <h4>1.1 &nbsp;&nbsp;&nbsp;Transformation</h4>
            <div CLASS="content">We transfer the M13 bacteria phage RF dsDNA into JM109 Escherichia coli, and incubate transferred E.coli on LB agar plates at 37℃ upside down for 12-16 hours.</div>
            <div CLASS="content">(1)Incubate competent cells on ice until it melts.</div>
            <div CLASS="content">(2)Add 4μg target DNA into 25uL cell suspension. Shake the centrifuge tubes slightly to mix the contents. Then incubate on ice for 30 minutes.</div>
            <div CLASS="content">(3)Put the centrifuge tube in 42℃ water for 90 seconds. Then quickly put it on ice for 2 minutes. Be careful not to shake the centrifuge tube.</div>
            <div CLASS="content">(4)Add 200uL LB medium into the centrifuge tube, then oscillating at 37 °C/180 rpm for 1 hour to revive the cells and make related gene express.</div>
            <div CLASS="content">(5)Streak adequate amount transformed cells (about 100uL) on the LB plate, incubate the LB agar plate at 37 °C upside down for 12-16 hours. (The LB plate was streaked with 40ul X-gal(20mg/ml) and 12ul IPTG (24mg/ml) before using. X-gal and IPTG were mixed before streaking)</div>
    

            <h4>1.2 &nbsp;&nbsp;&nbsp;Get M13 bacteriophage</h4>
            <div CLASS="content">We pick a blue single colony from LB agar plate. After the expanding culture of bacteria phage, we extract the bacteriophage.</div>
            <div CLASS="content">(1)Streak JM109 Escherichia coli cells (pick the blue one) on the pre-warmed LB agar plate to generate single colonies.</div>
            <div CLASS="content">(2)Incubate the LB agar plate overnight at 37 °C.</div>
            <div CLASS="content">(3)After overnight incubation, pick a blue single colony from the LB agar plate and inoculate it in a 50mL LB culture. Incubate for 8~12 hours at 37 °C/rpm.</div>
            <div CLASS="content">(4)Collect bacteria by centrifugation at 6,000 g for 20 minutes at 4°C.</div>
            <div CLASS="content">(5)Recover the supernatant (bacterial pellet can be discarded) and precipitate the M13 bacteria phage that bears the 7249-base scaffold by adding polyethylene glycol (average MW=8,000) and NaCl to final concentrations of 4% and 0.5 M, respectively.</div>
            <div CLASS="content">(6)Mix with magnetic stir bar until all PEG8000 has dissolved. CAUTION At this point, the supernatant should be a cloudy suspension. If it is still clear, it is likely that there is little or no phage present. Incubate it on ice for 30 minutes, then collect the precipitated bacteria phage by centrifugation at 6,000 g for 20 minutes.</div>
            <div CLASS="content">(7)Re-suspend pelleted bacteria phage in 3mL 1×TE buffer (10 mM Tris-HCl, 1 mM EDTA). This is "pre-inoculation" bacteria phage that will be used in following steps to scale up production of the M13mp18 scaffold strand.</div>
            <h4>1.3 &nbsp;&nbsp;&nbsp;Extract scaffold DNA—M13 DNA</h4>
            <div CLASS="content">We use E.Z.N.A.® M13 DNA Mini Kit to extract M13 ssDNA.</div>
            <div CLASS="content">(1)Transfer 1.5mL of the pre-obtained containing M13 bacteriophage into a fresh reaction tube.</div>
            <div CLASS="content">(2)Add 300μL MPG Buffer to the M13 supernatant and mix by vortex.</div>
            <div CLASS="content">(3)Let sit at room temperature for 10-15 minutes.</div>
            <div CLASS="content">(4)Add 700uL sample to a HiBind® M13 DNA Mini Column inserted into a 2 mL Collection Tube.</div>
            <div CLASS="content">(5)Centrifuge at 10,000 rpm for 30 seconds. Discard the filtrate and reuse the collection tube.</div>
            <div CLASS="content">(6)Repeat Steps4 and 5 until all the sample has been passed through the HiBind® M13 DNA Mini Column. (a fresh microcentrifuge tube can only be used for 1~3ml supernatant containing M13 bacteriophage)</div>
            <div CLASS="content">(7)Add 500mL MPB buffer into the column. Let it sit for 1 minute at room temperature. Then centrifuge it at 10000rpm for 30 seconds. Discard the filtrate and reuse the collection tube. Repeat this step.</div>
            <div CLASS="content">(8)Add 700uL SPW Wash Buffer. Centrifuge at 10,000 rpm for 30 seconds. (SPW Wash buffer concentrate is to be diluted with 80ml 100% ethanol to each bottle)</div>
            <div CLASS="content">(9)Discard the filtrate and reuse the Collection Tube. Repeat Steps 14-16 for a second SPW Wash Buffer wash step.</div>
            <div CLASS="content">(10)Centrifuge the empty column at maximum speed for 1 minute.</div>
            <div CLASS="content">(11)Insert the HiBind® M13 DNA Mini Column into a clean 1.5 mL microcentrifuge tube (not provided).</div>
            <div CLASS="content">(12)Add 30-50uL Elution Buffer heated to 60°C. (Make sure to add Elution Buffer directly onto the HiBind® M13 DNA Mini Column matrix)</div>
            <div CLASS="content">(13)Put the microcentrifuge tube in 60 °C water for 3~5 minutes.</div>
            <div CLASS="content">(14)Centrifuge at maximum speed for 1 minute. Store eluted DNA at -20 °C.</div>
            
            <h4>1.4 &nbsp;&nbsp;&nbsp;Verification of scaffold DNA</h4>
            <div CLASS="content">1.4.1 &nbsp;&nbsp;&nbsp; Measurement of the concentration and the purity of M13 DNA
               Use the nucleic acid analyzer to measure the concentration of M13 DNA. Use TE buffer to set zero. the samples are diluted by 20times.</div>
            <div CLASS="content">1.4.2 &nbsp;&nbsp;&nbsp;  Agarose gel electrophoresis</div>
            <div CLASS="content">(1)Dissolve 2 g of agarose in about 30mL of 1x TAE buffer. Use microwave to heat the liquid until it becomes clear and transparent.</div>
            <div CLASS="content">(2)When the liquid cools to about 60℃, add approximately 20μL ethidium bromide.</div>
            <div CLASS="content">(3)Poor it into electrophoretic apparatus, let it coagulate.</div>
            <div CLASS="content">(4)Each sample is loaded with loading buffer.</div>
            <div CLASS="content">(5)Run for 25-30min.</div>
            <div CLASS="content">(6)View gel on a UV table.</div>
            
            
            <h3>2.&nbsp;&nbsp;&nbsp;The folding process of DNA origami</h3>
            <h4>2.1&nbsp;&nbsp;&nbsp;Reaction condition</h4>
            <div CLASS="content">After a series of gradient experiment, we optimize the reaction condition: For a 50μL reaction system, the final concentration of every reagent:</div>
            <div CLASS="eimg50"><img SRC="images/exper/e1.png"></img></div>
          
            <h4>2.2&nbsp;&nbsp;&nbsp;PCR procedure</h4>
            <div CLASS="content">(1)Ring</div>
            <div CLASS="content">Lid Control Mode: reheating at 99°C.</div>
            <div CLASS="eimg40"><img SRC="images/exper/e2.png"></img></div>
            <div CLASS="efig">Figure 1: PCR program of ring</div>
            <div CLASS="content">(2)Rod</div>
            <div CLASS="content">Lid Control Mode: reheating at 99°C.</div>
             <div CLASS="eimg40"><img SRC="images/exper/e3.png"></img></div>
            <div CLASS="efig">Figure 2: PCR program of pole</div>
            <h4>2.3&nbsp;&nbsp;&nbsp;The purification and concentration of DNA origami</h4>
            <div CLASS="content">For improving the folding efficiency, the staple strands are added in a 10-fold higher concentration than the scaffold. For purification of the two monomers from the excess staple strands, a filtration step is carried out after folding. This is done by using Amicon Ultra-0.5 Centrifugal Filters for DNA purification and concentration with a pore size of 100 kD:</div>
            <div CLASS="content">(1)Insert the Amicon Ultra-0.5 device into one of the provided microcentrifuge tubes.</div>
            <div CLASS="content">(2)Add up to 100uL of sample and 400μl PCR buffer with Mg<sup>2+</sup> to the Amicon Ultra filter device.</div>
            <div CLASS="content">(3)Spin the device at 4500 × g for 15 minutes and discard the liquid in the microcentrifuge tube.</div>
            <div CLASS="content">(4)Repeat steps 2 and 3 for 4 times.</div>
            <div CLASS="content">(5)Place the filter device upside down in the microcentrifuge tubes, spin for 4 minutes at 1000 g to transfer the concentrated sample from the ultrafilter device to the tube.</div>
            <h4>2.4&nbsp;&nbsp;&nbsp;Loop the ring on the rod</h4>
            <div CLASS="content">Mix the ring and rod by 1:1, and incubate it at 37 for 30 minutes，then cool to 15°C by 3°C per minute.</div>
            <h3>3.&nbsp;&nbsp;&nbsp;The observation of DNA origami</h3>
            <h4>3.1&nbsp;&nbsp;&nbsp;TEM imaging</h4>
            <div CLASS="content">We use Transmission Electron Microscopy (TEM) to image the DNA origami structures (the ring, the rod, the ring looped on the rod). In this part, an additional DNA staining step is included, using a 2% uranyl acetate solution.</div>
            <div CLASS="content">(1)The plasma cleaned grids are clamped on with tweezers.</div>
            <div CLASS="content">(2)5μL of sample is applied on each grid and then incubate for 5-10 minutes.</div>
            <div CLASS="content">(3)The liquid is then removed from the grid using a filter paper.</div>
            <div CLASS="content">(4)For highly concentrated samples: an additional washing step is included using 5uL of ultrapure water.</div>
            <div CLASS="content">(5)For samples containing DNA: 4μL of 2% uranyl acetate is applied and directly removed using a filter paper. Afterwards a second drop (4μL) of uranyl acetate is applied and let stay on the grid for 10 seconds before removing it again.</div>
            <div CLASS="content">(6)The grids are dried for 20-30 minutes before they are stored in a grid sample holder.</div>
            <h4>3.2&nbsp;&nbsp;&nbsp;AFM imaging</h4>
            <div CLASS="content">We use atomic force microscopy (AFM) to observe structures of the DNA origami structures (the ring, the rod, the ring looped on the rod). We adopt coverslip as the sample preparation and tapping mode in fluid to observe the structure of the DNA origami structures and the coverslip surface has to be completely flat in order to achieve the best results.  </div>
            <div CLASS="content">(1)The coverslip is dipped in 0.1mg/mL poly-L-lysine (diluted with 1×PBS) overnight and dried in air.</div>
            <div CLASS="content">(2)Add 7uL of sample to the surface and incubate for 10 minutes.</div>
            <div CLASS="content">(3)After that, wash it with 100μL PCR buffer (10 times diluted folding buffer with a Mg<sup>2+</sup> concentration of 8mM for ring and 12mM for rod) twice and keep wet.</div>
            <h3 id="e10">4.&nbsp;&nbsp;&nbsp;Fluorescence detection</h3>
            <div CLASS="content">We conduct the FRET-experiment in a QuantaMaster™ 400 fluorescence spectrophotometer using a quartz cuvette under different temperature condition. Both the excitation slit and emission slit are 10 nm, and is measured at time scanning mode. By modifying FAM on the ring and DAB on the rod respectively, we can detect whether the ring has connected with the rod or not and can also monitor the movements of the ring. The fluorescence signal is collected continuously by the instrument, we use Felix GX software to get data.</div>
            <div CLASS="content">For the control, we first mix strands modified by FAM with strands modified by DAB to test whether normal FRET phenomenon occurs. FAM (donor) is excited at the illumination of 480nm and then emits fluorescence at the wavelength of 520nm(measured). Once FAM (donor) meets with DAB (quencher) modified on the rod in a narrow distance, FAM (donor) exhibits lower fluorescence while DAB (quencher) absorbs the whole emission light and leads to fluorescence quenching.</div>
            <div CLASS="eimg40"><img SRC="images/exper/e4.png"></img></div>
            <div CLASS="efig">Figure3: FRET phenomenon of single strands</div>
            <div CLASS="content"> When the ring has not been locked to the rod but diffuses freely in solution, the distance between FAM (on the ring) and DAB (on the rod) is too far to generate FRET phenomenon. Therefore, FAM emits high fluorescence.</div>
            <div CLASS="eimg40"><img SRC="images/exper/e5.png"></img></div>
            <div CLASS="efig">Figure4: Fluorescence of the ring</div>
            <div CLASS="content">Once the ring combines with the rod, FRET occurs so the intensity of fluorescence by FAM decreases.</div>
            <div CLASS="eimg40"><img SRC="images/exper/e6.png"></img></div>
            <div CLASS="efig">Figure5: Fluorescence of the ring looped in the pole</div>
            <div CLASS="content">We sustain one channel to obtain monochromatic light with changeable intensity. In the irradiation of 340nm at the temperature of 47.5°C, the duplex inserted with Azo dissociates while the distance between DAB and FAM elongates, resulting in the decrease of the intensity of fluorescence.</div>
            <div CLASS="content">Sequences in roadblock (on the rod) below are repeated from ref <div CLASS="reftexte1 reftext">[1]</div>.</div>
            <div CLASS="eimg40"><img SRC="images/exper/e7.jpg"></img></div>
            <div CLASS="eimg40"><img SRC="images/exper/e8.png"></img></div>
            <div CLASS="efig">Figure6: the ring looped in the pole after 30min-irradiation of 340nm</div>
            

<h2 id="e2">Result</h2>
            <h3>1.&nbsp;&nbsp;&nbsp;The Verification of Extracted M13mp18 DNA</h3>
            <div CLASS="content">After extracting DNA from M13 bacteriophage, we use the nucleic acid analyzer to measure the concentration and the purity of M13 DNA and do agarose gel electrophoresis to judge whether it can be applied to DNA origami procedure. Because the whole M13mp18 ssDNA has 7249 nucleotides and it is circular ssDNA, we suppose that the position of the M13mp18 ssDNA will be at approximately 3600bp.</div>
            <h4>1.1&nbsp;&nbsp;&nbsp;Measurement of the concentration and the purity of M13 DNA</h4>
            <div CLASS="content">In the measurement of the concentration and the purity of M13 DNA, we use 1×TE buffer to set zero and the M13 DNA we extract is diluted by 50 times with 1×TE buffer.</div>
            <div CLASS="eimg40"><img SRC="images/exper/e9.png"></img></div>
            <div CLASS="efig">Figure7 : Sample1-2: extracted M13mp18 DNA.</div>         
            <div CLASS="content">The results are shown in Figure 7. The value of A260/A280 is between 1.8 and 2.0 and the value of A260/A230 is between 2.0 and 2.5, which means the sample have little protein pollution and Carbohydrate pollution, it has a high purity as DNA.</div>
            <h4>1.2&nbsp;&nbsp;&nbsp;Agarose gel electrophoresis </h4>
            <div CLASS="content">In the agarose gel electrophoresis experiment, 0.7% agarose gels containing 0.1% ethidium bromide prepared at room temperature runs for 25 minutes at constant 110V and then is observed under UV.</div>
            <div CLASS="eimg40"><img SRC="images/exper/e10.png"></img></div>  
            <div CLASS="efig">Figure8 : Lane1-3: M13 DNA. Lane4-5: 10-time-dilued M13 DNA. Lane6: M13 DNA. Lane 7: M13 RF-DNA.</div>
            <div CLASS="content">The results are shown in Figure 8, which proves that the M13mp18 ssDNA we extract is about 3600bp.</div>
            <h3>2.&nbsp;&nbsp;&nbsp;The Observation of Structure</h3>
            <div CLASS="content">By optimizing the folding conditions, we successfully fabricate DNA origami ring and rod in a thermal ramp for 17 hours. We use TEM imaging, AFM imaging and agarose gel electrophoresis to characterize the origami outcome.</div>
            <h4>2.1&nbsp;&nbsp;&nbsp;The Observation of Ring</h4>
            <div CLASS="content">2.1.1&nbsp;&nbsp;&nbsp;Agarose gel electrophoresis</div>
            <div CLASS="content">In this experiment, 0.7% agarose gels containing 0.1% ethidium bromide prepared at room temperature runs for 25 minutes at constant 110V and then is observed under UV. We can analyze and compare the structural difference among samples from the migration speed of lanes.</div>
            <div CLASS="eimg30"><img SRC="images/exper/e11.png"></img></div> 
            <div CLASS="efig">Figure 9: Lane 1: DNA marker. Lane 2:M13 DNA. Lane 3: Ring</div>
            <div CLASS="content">The result is shown in Figure 9. The position of the band of the ring is higher than the position of the band of M13 DNA, which means we get an outcome that is different from M13 DNA.</div>
            <div CLASS="content">2.1.2&nbsp;&nbsp;&nbsp;AFM imaging</div>
            <div CLASS="content">The intact structure of ring is successfully folded after the origami experiment. Followed with a purification, we can finally see ring under AFM with a high yield.  In Figure 10D and Figure 10E, the height profile is given suggesting that the largest diameter of the ring is about 40nm, the smallest diameter of the ring is about 20nm, approaching our design 16nm and 35nm respectively, which confirms the successful folding. However, the height of the ring is about 3nm according to Figure 10F, which is lower than the height we expected, 10nm, we suppose that the structure of the ring is destroyed during the procedure of making the sample of the ring for AFM imaging.</div>
            <div CLASS="eimg50"><img SRC="images/exper/e12.png"></img></div>  
            <div CLASS="efig">Figure 10: AFM images of ring. (A, B) Wide field image of successfully assembled ring. (C) A magnified image of ring. (D, E) Height profile, curve between the vertical lines indicates the height of ring framed in image. (F) A model image corresponding to the real structure illustrated in image besides to give a vivid cognition of the successfully assembled ring</div>



            <div CLASS="content">2.1.3&nbsp;&nbsp;&nbsp;TEM imaging</div>
            <div CLASS="eimg60"><img SRC="images/exper/e13.png"></img></div>
            <div CLASS="efig">Figure 11: TEM image of the ring. (A) Wide field image of successfully assembled ring. (B, C) A magnified image of ring. (D) A model image corresponding to the real structure illustrated in image besides to give a vivid cognition of the successfully assembled ring</div>
            <div CLASS="content">Figure11 shows the ring we designed with a different model, which is a ring with 20nm external diameter, 15nm internal diameter and 5nm height. We can finally see a large sum of rings under TEM. The external diameter and internal diameter of the ring under TEM approaches our design respectively, which confirms the successful folding. However, we did not find too much rings in this form under TEM and the color of the ring is abnormal. Therefore, these rings could be bubbles.</div>
            <h4>2.2The observation of pole</h4>
            <div CLASS="content">2.2.1&nbsp;&nbsp;&nbsp;Agarose gel electrophoresis</div>
            <div CLASS="content">In this experiment, 0.7% agarose gels containing 0.1% ethidium bromide prepared at room temperature runs for 25 minutes at constant 110V and then is observed under UV. We can analyze and compare the structural difference among samples from the migration speed of lanes.</div>
            <div CLASS="eimg30"><img SRC="images/exper/e14.png"></img></div>
            <div CLASS="efig">Figure 12: Lane 1: M13 DNA. Lane 2: rod.</div>

            <div CLASS="content">The result is shown in Figure 12. The position of the band of the rod is higher than the position of the band of M13 DNA, which means we get an outcome that is different from M13 DNA.</div>
            <div CLASS="content">2.2.2&nbsp;&nbsp;&nbsp;AFM imaging</div>
            <div CLASS="eimg60"><img SRC="images/exper/e15.png"></img></div>
            <div CLASS="efig">Figure 13: AFM images of rod. (A) Wide field image of successfully assembled rod. (B) A magnified image of rod. (C,D) Height profile, curve between the vertical lines indicates the height of ring framed in image. (E) A model image corresponding to the real structure illustrated in image besides to give a vivid cognition of the successfully assembled rod</div>
            <div CLASS="content">The intact structure of rod is successfully folded after the origami experiment. Followed with a purification, we can finally see rod under AFM with a high yield.  In Figure 13C and Figure 13D, the height profile is given suggesting that the length of the rod is about 70nm, approaching our design 80nm, which confirms the successful folding.</div>
            <div CLASS="content">2.2.3&nbsp;&nbsp;&nbsp;TEM imaging</div>
            <div CLASS="eimg60"><img SRC="images/exper/e16.png"></img></div>
            <div CLASS="efig">Figure 14: TEM image of the ring. (A) Wide field image of successfully assembled rod. (B, C) A magnified image of rod. (D) Model images from different angles corresponding to the real structure illustrated in image besides to give a vivid cognition of the successfully assembled rod</div>
            <h4>2.3&nbsp;&nbsp;&nbsp;The observation of the ring looped in the pole </h4>
            <div CLASS="content">2.3.1&nbsp;&nbsp;&nbsp;Agarose gel electrophoresis</div>
            <div CLASS="content">In this experiment, 0.7% agarose gels containing 0.1% ethidium bromide prepared at room temperature runs for 25 minutes at constant 110V and then is observed under UV. We can analyze and compare the structural difference among samples from the migration speed of lanes.</div>
            <div CLASS="eimg30"><img SRC="images/exper/e17.png"></img></div>
            <div CLASS="efig">Figure 15: Lane1: ring. Lane2: rod Lane3: ring looped on rod.</div>
            <div CLASS="content">The result is shown in Figure 15. The position of the band of the ring looped on the rod is higher than the position of the band of ring and rod, which means we get an outcome that is different from ring or rod.</div>
            <div CLASS="content">2.3.2&nbsp;&nbsp;&nbsp;AFM imaging</div>
                        <div CLASS="content">we can finally see rod under AFM with a high yield. The height profile indicates that there is something in the front of the rod, which confirms successful combination between the ring and the rod.</div>
            <div CLASS="eimg60"><img SRC="images/exper/e18.png"></img></div>
            <div CLASS="efig">Figure16: AFM images of ring looped in rod. (A, D, E) Magnified images of ring looped in rod. (B, C) Height profile, curve between the vertical lines indicates the height of ring framed in image. (F) A model image corresponding to the real structure illustrated in image besides to give a vivid cognition of the successfully assembled ring looped in rod.</div>
            <div CLASS="content">From Figure 16C, we can find there is something with a height of 6nm in the front of the rod with the length of 100nm. The height of this thing approaches our designed height, which confirms the successful folding.</div>
            <h3>3.&nbsp;&nbsp;&nbsp;Fluorescence Detection</h3>
            <div CLASS="content">We modify FAM on the end of P1-1 of the ring and DAB on the end of P1' of the rod, which is complementary to P1-1 of the ring. Therefore, we can detect the occurrence of FRET if the ring loops on the rod. When we use 340nm laser to Irradiate the ring looped in the rod, the complementary strands will be untied for the characteristic of azobenzene. As our expects, the ring will go through the first door we design. As a result, we will detect the disappearance of FRET.</div>
            <div CLASS="content">We conduct the FRET-experiment in a QuantaMaster™ 400 fluorescence spectrophotometer using a quartz cuvette at room temperature about 25°C. The result is shown in Figure 17. When we detect the fluorescence intensity of the ring, the peak of wavelength is 520nm. Then, we detect the fluorescence intensity of the rod, and a disappearance of the peak in the wavelength on 520nm can be detected, which means the occurrence of FRET. After that, we use 340nm laser to Irradiate the ring looped on the rod for 15 minutes in room temperature, and we can detect an increase of the fluorescence intensity on 520nm compared to the ring looped on the rod without irradiation. When we use 340nm laser to Irradiate the ring looped in the rod for 30 minutes in room temperature, we can find another increase of the fluorescence intensity on 520nm compared to the ring looped in the rod with 15-min irradiation, which may indicate the movement of the ring.</div>
       
            <div CLASS="eimg60"><img SRC="images/exper/e20.png"></img></div>
            <div CLASS="efig">Figure 17: FRET measurement of the DNA origami: (A) Fluorescence changing of DNA origami monomer testing in fluorescence intensity-wavelength mode. (B, C) The intensity of 340nm laser.</div>
       
                      
<h2 id="e3">Materials</h2>           
            <h3>1.&nbsp;&nbsp;&nbsp;Machine</h3>
            <h4>1.1&nbsp;&nbsp;&nbsp;Principle of Agarose Gel Electrophoresis</h4>
            <div CLASS="content">Agarose-gel electrophoresis currently provides the most effective method available for high resolution separation of well-folded origami objects<div CLASS="reftexte2 reftext">[2]</div>. Agarose is a linear polysaccharide consisting of repeating units of D-galactose (β1-4)-linked to 3,6-anhydro-L-galactose that forms double helices during cooling, which associate into a three-dimensional network with gelling<div CLASS="reftexte3 reftext">[3]</div>. Separation of nucleic acids by agarose gel electrophoresis works by harnessing the negative charge of the phosphate backbone of nucleic acids. DNA molecules have a net negative charge spread evenly over their entire length so they will move through an agarose matrix in an electric field toward the positive rod. Shorter nucleic acids will be able to migrate through the matrix faster than larger ones during a given period of time. Depending upon the percentage of agarose used to make the gel, the range of linear separation will vary<div CLASS="reftexte4 reftext">[4]</div>.  Small agarose gel pores are crosslinked using high agarose density, and the resolution is high. In contrast, the agarose gel pores are larger using low agarose density, and the resolution is low<div CLASS="reftexte5 reftext">[5]</div>. DNA samples are stained with Ethidium bromide prior added to the agarose, fluorescing on insertion between DNA bases, so the bands can be well shown under the UV light. Principle of Atomic Force Microscopy</div>
            <h4>1.2&nbsp;&nbsp;&nbsp;Principle of Atomic Force Microscopy</h4>
            <div CLASS="content"> Atomic force microscopy was invented based on scanning tunneling microscopy. When the tip is scanning on objects, force between tip and atom outside the objects varies depending on their distance. At the same time, changing atomic force causes vibration and deformation of cantilever, further causing deflection of laser branch. So, distance between tip and objects can be calculated by determining displacement of laser, further concavity and convexity can be characterized and the atomic-scale topography of a surface can be mapped. </div>
            <h4>1.3&nbsp;&nbsp;&nbsp;Principle of Transmission Electron Microscopy</h4>
            <div CLASS="content"> Similar with optical microscope, transmission electron microscopy uses a beam of electrons transmitted to a specimen to form an image with an electromagnetic lens instead of the optical lens. Owing to the smaller de Broglie wavelength of electrons, it is capable of image formation at a significantly higher resolution, which is visual on a fluorescent screen. Transmission electron microscopy can characterize objects larger than 0.2nm and is widely used in DNA nanotechnology. </div>
            <h3>2.&nbsp;&nbsp;&nbsp;Attachment</h3>
            <div CLASS="content">(1)IPTG：Dissolve 238 mg IPTG (sigma I-6758) in 10 mL deionized water, store at -20°C</div>
            <div CLASS="content">(2)X-Gal(20mg/ml): Dissolve 2 g X-Gal in 100 mL DMF, store at -20°C.</div>
            <div CLASS="content">(3)1.0% agarose gel：Dissolve 0.2 g agarose in 20 mL 1× TAE buffer, heat the solution till agarose is completely dissolved, then add 20μL Ethidium bromide (V(EB):V(TAE)=1:1000) into the TAE buffer when it cools down around 60°C.</div>
            <div CLASS="content">(4)LB medium</div>
             <div CLASS="eimg40"><img SRC="images/exper/e21.png"></img></div>
            <div CLASS="content">(5)2x YT medium</div>
             <div CLASS="eimg30"><img SRC="images/exper/e22.png"></img></div>
            <div CLASS="content">(6)10×folding buffer: </div>  
            <div CLASS="eimg30"><img SRC="images/exper/e23.png"></img></div>
            
</div>
<div class="reference">
           <div class="rtitle">REFERENCE</div>
           <div class="rword">1.	Rothemund PW. Folding DNA to create nanoscale shapes and patterns. Nature. 2006;440(7082):297-302.</div>
           
           <div class="rword">2.Bellot G, McClintock MA, Lin C, Shih WM. Recovery of intact DNA nanostructures after agarose gel-based separation. Nat Methods. 2011 Mar;8(3):192-4</div>
           <div class="rword">3.Koontz L, et al. Agarose gel electrophoresis. Methods Enzymol, 2013;529:35-45.</div>
           <div class="rword">4.James D. Watson, Alexander Gann, Tania A. Baker, et al. Molecular Biology of Gene, 7th Edition.</div>
           <div class="rword">5.Nathalie Mignet, Daniel Scherman. DNA Complexes with Reducible Cationic Lipid for Gene Transfer. Methods in Enzymology Volume 373, 2003, Pages 357-369.</div>
</div>
 

<div STYLE="background:linear-gradient(to right,#D55757,#FFF); width:80%; height:3px; margin-left:50px; margin-top:100px;"></div>
<!--bottom-->
<div id="bottom">
        <div>Copyright©2018 HUST-CHINA All Rights Reserved</div>
       
</div>
</div>
</div>
</div>

<!--zhushi-->
<div id="ref_boxd1" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[1]Bandara HMD, Burdette SC. Photoisomerization in different classes of azobenzene. Chemical Society Reviews. 2012;41(5):1809-25.</div>
</div>
<div id="ref_boxd2" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[2]Nishioka H, Liang X, Kato T, Asanuma H. A photon-fueled DNA nanodevice that contains two different photoswitches. Angew Chem Int Ed Engl. 2012;51(5):1165-8.</div>
</div>
<div id="ref_boxp1" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[1]Rothemund PW. Folding DNA to create nanoscale shapes and patterns. Nature. 2006;440(7082):297-302.</div>
</div>
<div id="ref_boxp2" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[2]Shin J-S, Pierce NA. A Synthetic DNA Walker for Molecular Transport. Journal of the American Chemical Society. 2004;126(35):10834-5.</div>
</div>
<div id="ref_boxp3" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[3]Nishioka H, Liang X, Kato T, Asanuma H. A photon-fueled DNA nanodevice that contains two different photoswitches. Angew Chem Int Ed Engl. 2012;51(5):1165-8.</div>
</div>
<div id="ref_boxp4" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[4]Bhabha G, Johnson GT, Schroeder CM, Vale RD. How Dynein Moves Along Microtubules. Trends Biochem Sci. 2016;41(1):94-105.</div>
</div>
<div id="ref_boxp5" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[5]Wong CM, Wong KH, Chen XD. Glucose oxidase: natural occurrence, function, properties and industrial applications. Applied Microbiology and Biotechnology. 2008;78(6):927-38.</div>
</div>
<div id="ref_boxp6" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[6]Carlsson GH, Nicholls P, Svistunenko D, Berglund GI, Hajdu J. Complexes of Horseradish Peroxidase with Formate, Acetate, and Carbon Monoxide. Biochemistry. 2005;44(2):635-42.</div>
</div>

<div id="ref_boxm1" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[1]MECHANICS AND DYNAMIC BEHAVIORS OF DNA ORIGAMI docs.lib.purdue.edu/dissertations/AAI10268974/</div>
</div>
<div id="ref_boxm2" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[2]Alireza Sepehri. A mathematical model for DNA[J]. International Journal of Geometric Methods in Modern Physics, 2017, 14.</div>
</div>
<div id="ref_boxm3" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[3]López-Blanco J R, Chacón P. New generation of elastic network models[J]. Current Opinion in Structural Biology, 2016, 37:46-53.</div>
</div>
<div id="ref_boxm4" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[4]Zarai Y, Margaliot M, Tuller T. A deterministic mathematical model for bidirectional excluded flow with Langmuir kinetics[J]. Plos One, 2017, 12(8):e0182178.</div>
</div>
<div id="ref_boxm5" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[5]Garel T, Monthus C, Orland H. A simple model for DNA denaturation[J]. Epl, 2001, 55(1):132-138.</div>
</div>
<div id="ref_boxm6" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[6]You M, Chen Y, Zhang X, et al. An Autonomous and Controllable Light-Driven DNA Walking Device[J]. Angewandte Chemie International Edition, 2012, 124(10):2507-2510.</div>
</div>
<div id="ref_boxe1" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[1]Nishioka H, Liang X, Kato T, Asanuma H. A photon-fueled DNA nanodevice that contains two different photoswitches. Angew Chem Int Ed Engl. 2012;51(5):1165-8.</div>
</div>
<div id="ref_boxe2" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[2]Bellot G, McClintock MA, Lin C, Shih WM. Recovery of intact DNA nanostructures after agarose gel-based separation. Nat Methods. 2011 Mar;8(3):192-4.</div>
</div>
<div id="ref_boxe3" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[3]Koontz L, et al. Agarose gel electrophoresis. Methods Enzymol, 2013;529:35-45.</div>
</div>
<div id="ref_boxe4" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[4]James D. Watson, Alexander Gann, Tania A. Baker, et al. Molecular Biology of Gene, 7th Edition.</div>
</div>
<div id="ref_boxe5" CLASS="ref_box">
    <img CLASS="ref_img" SRC="images/reficon.png"></img>
    <div STYLE="width:100%; height:15px;"></div>
    <div CLASS="ref">[5]Nathalie Mignet, Daniel Scherman. DNA Complexes with Reducible Cationic Lipid for Gene Transfer. Methods in Enzymology Volume 373, 2003, Pages 357-369.</div>
</div>

<!--main menu-->
<div CLASS="menu">
      <div STYLE="width:100%; height:200px;"></div>
      <ul>
      <li id="project_menu">
         Project
      </li>
      
      <li id="design_menu">
         Design
      </li>
      <li id="experiment_menu">
         Experiment
      </li>
      
      <li id="team_menu">
         <a HREF="pages/team.htm">Team</a>
      </li>
      <li STYLE="color:#D55757">
         <a id="top" HREF="#firstpage">TOP</a>
      </li>
      </ul>    
</div>
<!--hidden layer-->
<div STYLE="width:100%; height:2px; background-color:white;"></div>

<div id="hidden_project" CLASS="hidden_res">
           <div STYLE="width:100%; height:2em;"></div>
           <div CLASS="menu_title"><a href="#project" class="p1">PROJECT</a></div>
           <div STYLE="width:100%; height:6em;"></div>    
           
           <div CLASS="menu_e">
           <a CLASS="p1" HREF="#p1"><img src="images/hidden/background.jpg"/></a>
           <h1 CLASS="submenu"><a CLASS="p1" HREF="#p1">Background</a></h1>
           <p><a CLASS="p1" HREF="#p1">DNA origami plays an indispensable role in constructing assembling spectacular micro-architectures and constructing molecular machines in nano world.
However, fuel strands contamination usually cannot be eliminated absolutely in system after chain reaction.</a></p>
           </div>
           
           <div CLASS="menu_e">
           <a CLASS="p1" HREF="#p2"><img src="images/hidden/idea.jpg"/></a>
           <h1 CLASS="submenu"><a CLASS="p2" HREF="#p2">Our idea</a></h1>
           <p><a CLASS="p2" HREF="#p2">We design a photo-controlled molecular motor, aiming to adopt a new power-input approach(light) to regulate the movement of nanomachine.</a></p>
           </div>          
                      
           
           <div CLASS="menu_e">
           <a CLASS="p1" HREF="#p4"><img src="images/hidden/furture.jpg"/></a>
            <h1 CLASS="submenu"><a CLASS="p4" HREF="#p4">Furture work</a></h1>
            <p><a CLASS="p4" HREF="#p4">In the future, we will apply this nanomachine to transport molecules like enzymes and hope to provide a more accurate way to describe the process of enzymes reaction.</a></p>
           </div>
     
             
</div>

<div id="hidden_design" CLASS="hidden_res">
           <div STYLE="width:100%; height:0.2em;"></div>
           <div CLASS="menu_title"><a class="d1" href="#design">DESIGN</a></div>  
           <div class="dbox">
           <div CLASS="menu_d">
           <a CLASS="d1" HREF="#d1"><img src="images/hidden/compound.jpg"/></a>
           <h1 CLASS="submenu"><a CLASS="d1" HREF="#d1">Compound</a></h1>
           <p><a CLASS="d1" HREF="#d1">Azobenzene (Azo) and its derivative 2'，6'-Me-Azobenzene(S-DMazo) have outstanding photo-isomerization properties</a>.</p>
           </div>
           
           <div CLASS="menu_d">
           <a CLASS="d2" HREF="#d2"><img src="images/hidden/protocols.jpg"/></a>
           <h1 CLASS="submenu"><a CLASS="d2" HREF="#d2">Compound wedged in DNA</a></h1>
           <p><a CLASS="d2" HREF="#d2">Wedge Azo or S-DMAzo into DNA strands respectively and optically-control the hybridization or dissociation of duplex.</a></p>
           </div>
           </div>
           
           <div class="dbox">          
           <div CLASS="menu_d">
           <a CLASS="d3" HREF="#d3"><img src="images/hidden/structure.jpg"/></a>
           <h1 CLASS="submenu"><a CLASS="d3" HREF="#d3">Structure</a></h1>
           <p><a CLASS="d3" HREF="#d3">We design a kind of light-driven nanomachine, consisting of a rod and ring in DNA origami.</a></p>
           </div>
           
           
           <div CLASS="menu_d">
           <a CLASS="d4" HREF="#d5"><img src="images/hidden/movement.jpg"/></a>
           <h1 CLASS="submenu"><a CLASS="d5" HREF="#d5">Movement</a></h1>
           <p><a CLASS="d4" HREF="#d5">The ring is motivated at different wavelengths of light along the rod.</a></p>
           </div>
           </div>
           
</div>


<div id="hidden_experiment" CLASS="hidden_res">
           <div STYLE="width:100%; height:2em;"></div>
           <div CLASS="menu_title"><a href="#experiment" class="e1">EXPERIMENT</a></div>
           <div STYLE="width:100%; height:6em;"></div>
           
           <div CLASS="menu_e">
           <a CLASS="e1" HREF="#e1"><img SRC="images/hidden/protocol.jpg"></img></a>
           <h1 CLASS="submenu"><a CLASS="e1" HREF="#e1">Protocol</a></h1>
           <p> <a CLASS="e1" HREF="#e1">In this part, we show our experiment plan including how to extraction scaffold, how to fold and observe our structure, how to verify the movement of the ring in the rod.</a></p>
           </div>
           
           <div CLASS="menu_e">
           <a CLASS="e2" HREF="#e2"><img SRC="images/hidden/result.png"></img></a>
           <h1 CLASS="submenu"><a CLASS="e2" HREF="#e2">Result</a></h1>
           <p> <a CLASS="e2" HREF="#e2">We observe the ring, the rod and the ring looped in the rod. Through the Fluorescence Detection, we realize the ring's movement through the first door in the rod.</a></p>
           </div>
           
           <div CLASS="menu_e">
           <a CLASS="e3" HREF="#e3"><img SRC="images/hidden/material.jpg"></img></a>
           <h1 CLASS="submenu"><a CLASS="e3" HREF="#e3">Material</a></h1>
           <p> <a CLASS="e3" HREF="#e3">The introduction of the machine and the  reagents we used in the experiment</a></p>
           </div>
</div>

<div id="hidden_simulation" CLASS="hidden_res">
           <div CLASS="menu_title">Simulation</div>
           
           <div CLASS="menu_s">
           <h1 CLASS="submenu"><a CLASS="s1" HREF="#s1"></a></h1>
           <p></p>
           </div>
           
           <div CLASS="menu_s">
           <h1 CLASS="submenu"><a CLASS="s2" HREF="#s2"></a></h1>
           <p></p>
           </div>
</div>



</body>
</html>

























